packages/@lwc/engine-core/src/framework/rendering.ts

```diff
@@ -482,7 +483,7 @@ function unmountVNodes(
 }
 
 function isVNode(vnode: any): vnode is VNode {
-    return vnode != null;
+    return vnode != null && !isPlaceholderVFragment(vnode);
```

Comment:
This line is very confusing. This method is to test if an object is a vnode... vfragments are vnodes, isn't it?

Reply:
Yes, you're correct. I mentioned in the comment above that if we go with this approach, this function will need renaming and maybe some refactoring.